src/Command/Setup.php

```diff
@@ -75,7 +75,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         return 0;
     }
 
-    private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata)
+    private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata) : \Generator
```

Comment:
There is no convention yet about this AFAIK, but I personally prefer:

```diff
- private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata) : \Generator
+ private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata): \Generator
```

(no space between the closing parentheses and the colon)

WDYT ?